common/persistence/nosql/nosqlplugin/cassandra/queue.go

```diff
@@ -183,7 +183,7 @@ func (db *cdb) InsertQueueMetadata(
 	query := db.session.Query(templateInsertQueueMetadataQuery, queueType, clusterAckLevels, version).WithContext(ctx)
 	_, err := query.ScanCAS()
 	if err != nil {
-		return fmt.Errorf("failed to insert initial queue metadata record: %v, Type: %v", err, queueType)
+		return err
```

Comment:
Nit: I thought we want to keep some information about the error like queueType, why should we remove the wrapping?

Reply:
The error information is still preserved in the caller of this function where we do error conversion. Return raw error here so that the caller in cassandraQueue.go can do the conversion, otherwise it won't be able to tell the difference between errors.